File: Codechef/VCS.cpp
//https://www.codechef.com/problems/VCS

#include <iostream>
using namespace std;

int main()
{
    int t;
    cin >> t;
    while (t--)
    {
        int n, m, k;
        cin >> n >> m >> k;
        int ignore[m], track[k], i, x = 0, y = 0;
        int a[n + 1] = {0};
        for (i = 0; i < m; i++)
        {
            cin >> ignore[i];
            a[ignore[i]]++;
        }
        for (i = 0; i < k; i++)
        {
            cin >> track[i];
            a[track[i]]++;
        }
        for (i = 1; i <= n; i++)
        {
            if (a[i] == 0)
            {
                y++;
            }
            else if (a[i] == 2)
            {
                x++;
            }
        }
        cout << x << " " << y << endl;
    }
    return 0;
}

File: Codechef/VCS.py
for _ in range(int(input())):
    n,m,k=map(int,input().split())
    a=list(map(int,input().split()))
    b=list(map(int,input().split()))
    f=0;l=0
    for i in range(1,n+1):
        if i in a and i in b:
            f+=1
        elif i not in a and i not in b:
            l+=1
    print(f,l)

# Click on this link to solve:
# https://www.codechef.com/problems/VCS?tab=statement